cps/src/main/java/org/jenkinsci/plugins/workflow/cps/steps/LoadStepExecution.java

```diff
@@ -56,4 +71,20 @@ public void stop(Throwable cause) throws Exception {
 
     private static final long serialVersionUID = 1L;
 
+    /** Allows content of the loaded script to be substituted. */
+    @Restricted(NoExternalUse.class) // for now anyway
+    public interface Replacer extends ExtensionPoint {
```

Comment:
I'm wondering why our replacer does a *single* replacement rather than taking a Map<String,String> of replacements or similar?